src/main/java/seedu/address/logic/commands/EditAppointmentCommand.java

```diff
@@ -90,7 +90,8 @@ public CommandResult execute(Model model) throws CommandException {
 
         // get the appointment to edit from the list of appointments of Person
         Appointment appointmentToEdit = appointmentList.get(appointmentIndex.getZeroBased());
-        Appointment editedAppointment = createEditedAppointment(appointmentToEdit, editAppointmentDescriptor);
+        Appointment editedAppointment = createEditedAppointment(appointmentToEdit, editAppointmentDescriptor,
+                editedPerson);
 
         editedPerson.editAppointment(appointmentIndex.getZeroBased(), editedAppointment);
```

Comment:
currently after edit_appt command, FE does not update. Same situation for delete_appt